src/QMCTools/QMCGaussianParserBase.cpp

```diff
@@ -2408,6 +2411,15 @@ xmlNodePtr QMCGaussianParserBase::createMultiDeterminantSetFromH5()
   xmlNewProp(detlist, (const xmlChar*)"neb", (const xmlChar*)cineb.str().c_str());
   xmlNewProp(detlist, (const xmlChar*)"nstates", (const xmlChar*)nstates.str().c_str());
   xmlNewProp(detlist, (const xmlChar*)"cutoff", (const xmlChar*)ci_thr.str().c_str());
+  if (nbexcitedstates >= 1)
+  {
+    app_log() << "WARNING!! THE HDF5 Contains CI coefficients for " << nbexcitedstates - 1
```

Comment:
please update "nbexcitedstates - 1" to "nbexcitedstates" and also add "excited states."